.github/workflows/pr-check.yml

```
@@ -9,18 +12,22 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
+    env:
+      ENV: .env
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
       - name: Set up Node.js
         uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)